app/assets/stylesheets/base/layout.scss

```diff
@@ -36,6 +36,11 @@
   grid-template-columns: var(--layout);
   padding: var(--layout-padding);
 
+  &__registration {
+    max-width: 100%;
+    background-color: white;
```

Comment:
(nitpick, non-blocking): Could we use the utility class `bg-base-inverted` and `max-w-100` here?

Reply:
Done.